src/routes/tools/annual-averages/_data.js

```diff
@@ -55,154 +25,196 @@ const convertToAnnual = (values) => {
   });
 };
 
-export async function addModel(config, params, method, modelId) {
+/**
+ * The following 3 functions take the list of observed, models and ensemble
+ * raster series and add extra props. These props describe how the data
+ * should be fetched from the API or how the data is shown in the charts.
+ * List of extra props:
+ * slugs - list of raster series names in API
+ * mark - describes how the timeseries will be displayed in the chart (line/area)
+ * visible - controls whether line/area in chart is hidden/shown
+ *      when user clicks on corresponding legend key. Note: The stats component
+ *      also uses this prop to include/remove series from summary calculations
+ * @param {object} config
+ * @return {array}
+ */
+
+// The observed data is usually a single raster series, so only 1 slug
+const getObservedSeries = (config) => {
```

Comment:
You can also simplify this by de-structuring in the param, e.g.

```js
const getObservedSeries = ({ climvarId }) => OBSERVED.map((d) => { ... })
```

Reply:
Fixed via https://github.com/berkeley-gif/caladapt-website-2021/commit/34320136ca64c3586e3ac3edc328993cd1c9ed09